app/src/main/java/com/gholem/moneylab/features/chart/viewmodel/ChartViewModel.kt

```diff
@@ -22,7 +23,9 @@ class ChartViewModel @Inject constructor(
     private val getTransactionListUseCase: GetTransactionListUseCase
 ) : ViewModel() {
 
-    private val LAST_MONTH = 2592000000L
+    val LAST_MONTH = 2678400000L
+    var COUNT_MONTH = 0L
+    private val TIME_IN_MILI = 1000000000L
```

Comment:
If this is unnecessary in given case (as I think you just assigned some random value for `TransactionModel`) then you don't need any new field, just set `0L` in constructor